### ISSUE TITLE
milksnake long description markdown is not properly rendered on pypi

### ISSUE THREAD

**Issue by kwlzn** [getsentry/milksnake#18]
seen here: https://pypi.org/project/milksnake/#description

![image](https://user-images.githubusercontent.com/1883523/41308249-44c04096-6e30-11e8-8658-3b22691a7b6e.png)

per the internet:

```
As of March 16, 2018, PyPI.org aka Warehouse (finally) supports Markdown in long descriptions. Warehouse replaced the old legacy PyPI implementation in April 2018.

You need to:

 - Make sure setuptools is upgraded to version 38.6.0 or newer
 - Make sure twine is upgraded to version 1.11.0 or newer
 - Make sure wheel is upgraded to version 0.31.0 or newer
 - Add a new field named long_description_content_type to your setup() call, and set it to 'text/markdown':

setup(
    long_description="""# Markdown supported!\n\n* Cheer\n* Celebrate\n""",
    long_description_content_type='text/markdown',
    # ....
)

- Use twine to upload your distribution to PyPI.
```

**Comment by kwlzn** [comment]
a super quick PR for the setup.py change here: https://github.com/getsentry/milksnake/pull/19

**Comment by mitsuhiko** [comment]
This should be fixed with #19 I believe.
